### DISCUSSION TITLE
support for charts and dashboards

### DISCUSSION THREAD

**Discussion by HaveF** [datasette/datasette-comments#16] [Ideas]
Hi, :-)

This plugin is very interesting!

I'm wondering if it's possible to make it possible to support not only data, but also all kinds of charts, dashboards, etc., after all, datasette doesn't just present tables

**Comment by asg017** [comment]
Thanks for sharing! That makes sense, though commenting on charts will require some coordination between datasette-comments and individual visualization plugins. 

Though I am working on a new visualization Datasette plugin, [datasette-plot](https://github.com/datasette/datasette-plot), so I'll give them some more thought!

**Reply by HaveF** [reply]
Hi, @asg017 ,thanks for your reply!

There is a great dashboard plugin for your reference -- [rclement/datasette-dashboards](https://github.com/rclement/datasette-dashboards). If your plugin have some APIs, these type of plugins might be easier to integrate the comments functions
